backend/fms_core/migrations/0019_v3_1_2.py

```diff
@@ -0,0 +1,76 @@
+from django.conf import settings
+from django.db import migrations
+from django.contrib.auth.models import User
+import reversion
+import datetime
+
+ADMIN_USERNAME='biobankadmin'
+
+
+def fix_process_data(apps, schema_editor):
+    Version = apps.get_model("reversion", "Version")
+    Revision = apps.get_model("reversion", "Revision")
+    ProcessSample = apps.get_model("fms_core", "ProcessSample")
+    Process = apps.get_model("fms_core", "Process")
+
+    affected_revisions_ids = Revision.objects.filter(date_created__range=["2021-04-29", "2021-05-01"]).values_list('id', flat=True)
```

Comment:
Django complains about these dates not being timezone aware (warning), not sure if this is worth acting on it though.

Reply:
Dont think it's worth the trouble of making it timezone aware. The date filter does not have to be super precise, it was mostly just to not have to go through all the ever existing revisions :P  and it succeeds at filtering the right data  :)